duckduckgo.safariextension/js/oninstall.js

```diff
@@ -0,0 +1,24 @@
+/*
+   Copyright (C) 2016 DuckDuckGo, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+"use strict";
+
+setTimeout(function() {
```

Comment:
We want this to run on the first page load after they install the extension right? Did you try adding a handler for DOMContentLoaded? 

```
document.addEventListener('DOMContentLoaded', function() {
  // check for atb here
}, false);
```

That might still not be enough though to get it to run after the internal code that also runs on that same event.